db/seeds.rb

```diff
@@ -1,4 +1,29 @@
-g1 = Giraffe.find_or_create_by!(name: "steven", description: "he's a cool giraffe, but he thinks people are leaves, so he wont stop biting them, which is proving to be problematic in a myriad of ways")
-g2 = Giraffe.find_or_create_by!(name: "giraffey", description: "Just like the default giraffe")
-g3 = Giraffe.find_or_create_by!(name: "bobby", description: "A very odd giraffe, deer shaped and also brown. not sure if he's actually a giraffe")
-g4 = Giraffe.find_or_create_by!(name: "sarah", description: "she's a kind giraffe, who loves running around the Savanna")
\ No newline at end of file
+
+User.destroy_all
+
+u1 = User.new(username: "Ted Francis", email: "ted@fake.com")
+u1.password =  "1234567"
```

Comment:
To make these test user accounts stronger, you could create passwords in your ENV file and use the ENV variable for the passwords so they are protected, even though they're fake